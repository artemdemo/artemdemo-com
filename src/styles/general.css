/*
 * In a nested list markdown is adding <p> tag for the parent.
 * It's adding gap, which I want to eliminate
 */
li > p:first-child {
    margin-bottom: 0;
}

/*
 * Code snippets are taking too much space.
 * It's only visually but it looks like font size there is bigger than the rest of the text.
 * Therefore I'm reducing it.
 */
.gatsby-highlight {
    font-size: 0.9rem;
}

blockquote {
    padding: 0.2rem 0 0 1rem;
    border-left: 4px solid #e8e8e8;
}
