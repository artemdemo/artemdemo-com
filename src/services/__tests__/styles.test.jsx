import React from 'react';
import renderer from 'react-test-renderer';
import * as styles from '../styles';

describe('styles', () => {
    it('resetButton', () => {
        const tree = renderer.create(
            <div>
                {styles.resetButton}
            </div>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });

    it('resetUl', () => {
        const tree = renderer.create(
            <div>
                {styles.resetUl}
            </div>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });

    it('clearFix', () => {
        const tree = renderer.create(
            <div>
                {styles.clearFix}
            </div>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});