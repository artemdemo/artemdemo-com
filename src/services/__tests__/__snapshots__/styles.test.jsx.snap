// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styles clearFix 1`] = `
<div>
  
    &::after {
        content: '';
        display: table;
        clear: both;
    }

</div>
`;

exports[`styles resetButton 1`] = `
<div>
  
    border: none;
    margin: 0;
    padding: 0;
    width: auto;
    overflow: visible;

    background: transparent;

    cursor: pointer;

    /* inherit font & color from ancestor */
    color: inherit;
    font: inherit;

    /* Normalize \`line-height\`. Cannot be changed from \`normal\` in Firefox 4+. */
    line-height: normal;

    /* Corrects font smoothing for webkit */
    -webkit-font-smoothing: inherit;
    -moz-osx-font-smoothing: inherit;

    /* Corrects inability to style clickable \`input\` types in iOS */
    -webkit-appearance: none;

</div>
`;

exports[`styles resetUl 1`] = `
<div>
  
    list-style-type: none;
    margin: none;
    margin: 0;
    padding: 0;

</div>
`;
